ember-toucan-core/src/components/form/textarea-field.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-col">
+  <Form::Field as |field|>
+    <field.Label for={{field.id}} data-textarea-label>{{@label}}</field.Label>
+
+    {{#if @hint}}
+      <field.Hint id={{field.hintId}} data-textarea-hint>{{@hint}}</field.Hint>
+    {{/if}}
+
+    <field.Control>
+      <textarea
+        aria-invalid={{if @error "true"}}
+        aria-describedby="{{if @error field.errorId}} {{if @hint field.hintId}}"
+        class="block h-20 min-h-6 p-1 bg-overlay-1 focus:outline-none rounded-sm shadow-focusable-outline focus:shadow-focus-outline
+          {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
+        disabled={{@isDisabled}}
+        id={{field.id}}
+        ...attributes
+      />
+    </field.Control>
+
+    {{#if @error}}
+      <field.Error
+        id={{field.errorId}}
+        data-textarea-error
+      >{{@error}}</field.Error>
```

Comment:
This assumes `@error` is always a simple (escaped) string, no further HTML markup. Which is not wrong, if that is what we want.

headless-form does allow for more than one validation error though. In terms of UX we can decide to
* only show the first error (which is a valid approach IMO, and would work fine with the current implementation here)
* show all errors, rendered with `<br/>` or a full `<ul>` list maybe? In which case we wouldn't be able to support this with a simple `@error` I think?

To support the second case (if we want that), we could
* conditionally yield to a named block, like `{{#if (has-block "error")}}{{yield to="error"}}{{(/if}}`, letting the user decide how to render the list of errors
* or let `@error` also be an array, detect that here, and `{{#each}}` over them here, with some opinionated way to render them (`<br/>` vs. `<ul>` vs. whatever). Which is ok (maybe better), since we are representing a design system here, so can impose the way it should look

Thoughts?

Reply:
> conditionally yield to a named block, like {{#if (has-block "error")}}{{yield to="error"}}{{(/if}}, letting the user decide how to render the list of errors

I'm glad you mentioned this! I was thinking even yielding to our other blocks as well, hint and label to allow a bit more flexibility with things.  What do you think?  I like this idea though, exposing a block for errors.

> or let @error also be an array, detect that here, and {{#each}} over them here, with some opinionated way to render them (<br/> vs. <ul> vs. whatever). Which is ok (maybe better), since we are representing a design system here, so can impose the way it should look

I personally quite like this idea!  I'm curious from a design perspective how multiple errors could look though.  Maybe I'll ask in our channel about it and see what folks think then go from there?

Reply:
Just met with the team - we are going to wait until David is back on Monday to decide how we want to handle this.  I went ahead and filed https://github.com/CrowdStrike/ember-toucan-core/issues/44 for tracking and will pick up as soon as we get an answer!  It sounded as though we want to support what you described though - just need to figure out how it should look